.circleci/config.yml

```diff
@@ -100,10 +100,10 @@ commands:
     steps:
       - run:
           name: Download flutter SDK
-          command: git clone -b stable https://github.com/flutter/flutter.git flutter
+          command: git clone -b stable https://github.com/flutter/flutter.git ../flutter
```

Comment:
This was causing the repo to be dirty

Reply:
We could also add the flutter directory to gitignore. I kinda prefer that approach to keep everything in the same workplace.

Reply:
ohhhh good point! haha